### PR TITLE
Parallel routes initial commit

### PR DESCRIPTION
**Pull request by SathyaShreeRajkumar**
Initial Project Setup with Parallel routing

### CODE REVIEW

File: nextjs_tdd/src/app/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import './globals.css'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata: Metadata = {
+    title: 'Create Next App',
+    description: 'Generated by create next app'
```
Comment (aashikvilla) [inline nextjs_tdd/src/app/layout.tsx]: Either remove it or update with appropriate values

File: nextjs_tdd/src/app/@airlines/page.tsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { COMMON_CONST } from '@/constants/app-constants'
+
+export default function AirlinesPage() {
+    return <div>{COMMON_CONST.CHOOSE_TO_FLY}</div>
```
Comment (aashikvilla) [inline nextjs_tdd/src/app/@airlines/page.tsx]: Having text in div is acceptable, but try to use semantic HTML elements whenever possible to improve the accessibility and readability of your code. 
eg., p or h1 to h6 or span, etc

File: nextjs_tdd/src/app/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    airlines: React.ReactNode
+}>) {
+    return (
+        <html lang="en">
+            <body className={inter.className}>
+                {children}
+                {home}
+                {airlines}
+            </body>
+        </html>
```
Comment (aashikvilla) [inline nextjs_tdd/src/app/layout.tsx:24]: Can we have tests for layout as well

File: nextjs_tdd/src/app/layout.tsx (hunk 1 above)
Comment (SathyaShreeRajkumar) [reply nextjs_tdd/src/app/layout.tsx]: Updated with appropriate value.

File: nextjs_tdd/src/app/@airlines/page.tsx (hunk 2 above)
Comment (SathyaShreeRajkumar) [reply nextjs_tdd/src/app/@airlines/page.tsx]: Used HTML elements.

File: nextjs_tdd/src/app/layout.tsx (hunk 3 above)
Comment (SathyaShreeRajkumar) [reply nextjs_tdd/src/app/layout.tsx:24]: Updated testcases.

**Review by aashikvilla — APPROVED** [review]
You can merge this PR

**Pull request merged**